pkg/addr/host_spec.gobra

```diff
@@ -19,49 +19,61 @@ package addr
 import (
 	"net"
 
-	. "github.com/scionproto/scion/verification/utils/definitions"
 	"github.com/scionproto/scion/verification/utils/slices"
 )
 
 pred (h HostNone) Mem() { len(h) == HostLenNone }
-pred (h HostNone) LowMem() { true }
-
-HostNone implements HostAddr {
-	(h HostNone) String() (str string) {
-		unfold acc(h.Mem(), R13/2)
-		str = h.String()
-		fold acc(h.Mem(), R13)
-	}
-}
+
+ghost
+requires h.Mem()
+decreases
+pure func (h HostNone) IsLow() bool
```

Comment:
please mark all of these as `trusted` so that it is easy to find them once we want to add the body